addons/l10n_in/tests/common.py

```diff
@@ -0,0 +1,131 @@
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+class L10nInTestInvoicingCommon(AccountTestInvoicingCommon):
+    @staticmethod
+    def setup_common_india(clss):
```

Comment:
1. it's not possible to convert it to a classmethod because in that case we could only call it from `L10nInTestInvoicingCommon` and its children and not from the `L10nInTestAccountReportsCommon` class

It's not possible for `L10nInTestAccountReportsCommon` to inherit from `L10nInTestInvoicingCommon` because it and its children require data/functions declared in `TestAccountReportsCommon`.

Reply:
I tried that and it doesn't work (the tests just don't run/get stuck somewhere), i think its because both `TestAccountReportsCommon` and `L10nInTestInvoicingCommon` inherit from `AccountTestInvoicingCommon` somewhere down the line, which means that the setUpClass functions of `AccountTestInvoicingCommon` and all its parents will be called twice. I suspect this is the reason it does'nt work

Reply:
Yeah it works, weird, i must have done something wrong when i tried it thx!